usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction.md

```diff
@@ -208,51 +208,128 @@ Returns transaction records which match the keyword and award type code filters.
 
 # Data Structures
 
-## PageMetaDataObject (object)
+## Request Objects
+
+### PageMetaDataObject (object)
 + `page`: 1 (required, number)
 + `hasNext`: false (required, boolean)
 + `hasPrevious`: false (required, boolean)
 + `next` (required, number, nullable)
 + `previous` (required, number, nullable)
 
-## TransactionResponse (object)
+### AdvancedFilterObject (object)
```

Comment:
Are we potentially leaving out some filters? I am referring to "legal_entities" and or "recipient_id".

Reply:
I checked the codebase and tried hitting the API. These filters do not seem to be supported by the endpoint.